src/common/translation.cpp

```diff
@@ -127,7 +127,7 @@ wxString GetPreferredUILanguageFallback(const wxArrayString& WXUNUSED(available)
 
 #ifdef __WINDOWS__
 
-wxString GetPreferredUILanguage(const wxArrayString& available)
+wxString GetPreferredUILanguage(const wxArrayString& available, wxArrayString &allPreferred)
```

Comment:
Very minor, but could you please apply this:

```suggestion
wxString GetPreferredUILanguage(const wxArrayString& available, wxArrayString& allPreferred)
```